docs/api-docs/catalog/price-list-overview.md

```diff
@@ -4,6 +4,7 @@
 	<ul>
         <li><a href="#pricelist_what-is-a-pricelist">What is a Price List?</a></li>
         <li><a href="#pricelist_definitions">Definitions</a></li>
+        <li><a href="#pricelist_assignments">Assignments</a></li>

```

Comment:
This is anchor won't  work. href should be: `#price-list-assignments`  (lowercase h2 text separated by `-`).  Use markdown all in one vs code extension to generate ToC and re-submit the pr.